### DISCUSSION TITLE
How to set a default redis connection ?

### DISCUSSION THREAD

**Discussion by ghostlexly** [taskforcesh/bullmq#2052] [Q&A]
Hey,

I'm always passing the redis connection parameters on my queues and workers.

Actual code:
```ts
const facebookQueue = new Queue("facebook", {
  connection: REDIS_CONNECTION,
});
```

Expected:
```ts
const facebookQueue = new Queue("facebook");
```

How i can set the default connection somewhere so i don't have to set the connection each time?

**Comment by manast** [comment]
You can't do that, that would not be a good development practice in any case.

**Reply by ghostlexly** [reply]
Why?
It's the same Redis connection for all of my queues in my case.

**Reply by manast** [reply]
Then create a factory method and use that instead. We are not going to change this.
